src/components/genre-select/GenreSelect.css

```diff
@@ -0,0 +1,66 @@
+.containerGenreSelect {
+    width: 100%;
+    background-color: #232323;
+    padding: 50px;
+}
+.inputLabel {
+    display: flex;
+    color: #F65261;
+    font-family: Montserrat;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: normal;
+    letter-spacing: 0.889px;
+    text-transform: uppercase;
+    text-align: left;
+    margin: 0 10px;
+}
+.selectGenre {
+    width: 525px;
+    height: 57px;
+    flex-shrink: 0;
+    border-radius: 4px;
+    opacity: 0.8031;
+    background: #323232F2;
+    border-color: transparent;
+    text-align: left;
+} ul {
+      border-radius: 4px;
+      background: rgba(35, 35, 35, 0.92);
+      box-shadow: 0 10px 50px 0 rgba(0, 0, 0, 0.10), 0 10px 20px 0 rgba(0, 0, 0, 0.10), 0 2px 10px 0 rgba(0, 0, 0, 0.10), 0 0 2px 0 rgba(0, 0, 0, 0.10);
+      backdrop-filter: blur(14px);
+  }
+  label {
+      color: white;
+  }
+.menuItemGenre {
+    color: #FFF !important;
+    font-family: Montserrat !important;
+    font-size: 20px !important;
+    font-style: normal !important;
+    font-weight: 400 !important;
+    line-height: normal !important;
```

Comment:
Just like a reminder, Try to avoid using `!important` in CSS files.

"Using the !important rule in CSS is generally considered to be bad practice because it overrides all other styles, regardless of their specificity. This can make your stylesheets difficult to maintain and can cause unexpected behavior in your website"
[Source](https://element.how/why-you-should-avoid-using-important-in-css/#:~:text=Using%20the%20!,unexpected%20behavior%20in%20your%20website.)